common/metrics/otel_metrics_handler_test.go

```diff
@@ -190,6 +195,7 @@ func TestMeter(t *testing.T) {
 						Min:          metricdata.NewExtrema[int64](int64(testBytes)),
 						Max:          metricdata.NewExtrema[int64](int64(testBytes)),
 						Sum:          int64(testBytes),
+						Exemplars:    []metricdata.Exemplar[int64]{},
```

Comment:
@yiminc just FYI I had to fix these unit tests because what used to be `nil` is now an empty list.